packages/fast-form-generator-react/src/form/form-item.base.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import SoftRemove from "./soft-remove";
```

Comment:
This isn't common terminology I'm familiar with on the web?  Maybe a comment where this is first used may help explain what soft remove is?

Reply:
Sounds good, added to the component description.